searx/engines/__init__.py

```diff
@@ -44,6 +44,9 @@
     "display_error_messages": True,
     "tokens": [],
 }
+# set automatically when an engine does not have any tab category
+OTHER_CATEGORY = 'other'
```

Comment:
Nit: would `NON_CATEGORY = 'uncategorized'` better?

Reply:
I don't think so. Firstly "uncategorized" would be completely misleading since engines with categories can be in the "OTHER_CATEGORY". Secondly naming the variable something other than the display text just makes the mental mapping more complicated.